dev-docs/bidders/consumable-server.md

```diff
@@ -0,0 +1,373 @@
+
+The Consumable adapter requires setup and approval from your Consumable account manager, even for existing Consumable publishers. Please reach out to your account manager to enable Prebid for your account.
+
+### Disclosure
+
+The Consumable bid adapter may cycle the ad initially shown with a new one at various intervals. This means the advertiser meta-data accompanying the bid response may be incomplete at the time of response.
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name        | Scope    | Description                    | Example | Type      |
+|-------------|----------|--------------------------------|---------|-----------|
+| `siteId`    | required | The site ID from Consumable.    | `12345` | `integer` |
+| `networkId` | required | The network ID from Consumable. | `9969`  | `integer` |
+| `unitId` | required | The unit ID from Consumable. | `987654`  | `integer` |
+| `unitName` | required | The unit name from Consumable. | `cnsmbl-unit`  | `string` |
+| `placementId` | required | the placementid from Consumable. | `0421008445828ceb46f496700a5fa65e` | `string` |
+
+## Table of contents
+
+* [Table of contents](#table-of-contents)
+* [Introduction](#introduction)
+* [Supported media types](#supported-media-types)
+* [Setup instructions to call Consumable through Prebid Server](#setup-instructions-to-call-consumable-through-prebid-server)
+  * [Call Consumable from a web browser](#call-consumable-from-a-web-browser)
+  * [Call Consumable from Prebid Mobile SDK](#call-consumable-from-prebid-mobile-sdk)
+  * [Call Consumable from CTV/long-form video environment](#call-consumable-from-ctvlong-form-video-environment)
+* [Bid request parameters](#bid-request-parameters)
+  * [Site](#site)
+  * [App](#app)
+* [Examples](#examples)
+
+<a id="introduction"></a>
+
+## Introduction
+
+Publishers can use Prebid Server in any of the following ways with Consumable. Consumable's adapter supports all of the following methods:
+
+* If you want to call Consumable from a web environment, you can use Prebid.js to call Prebid Server, and then Prebid Server uses our server-side adapter to call Consumable. This reduces workload on the browser. For set up instructions, see [Call Consumable from a web browser](#call-consumable-from-a-web-browser) section on this page.
+* In mobile apps, you can use the Prebid Mobile SDK to call Prebid Server and then Prebid Server uses our server-side adapter to call Consumable. For set up instructions, see [Call Consumable from Prebid Mobile SDK](#call-consumable-from-prebid-mobile-sdk) section on this page.
+* In CTV apps and other long-form video environments, you (or the SSAI vendor) can make a call to Prebid Server using OpenRTB, and then Prebid Server uses our server-side adapter to call Consumable. For set up instructions, see [Call Consumable from CTV/long-form video environment](#call-consumable-from-ctvlong-form-video-environment) section on this page.
+
+<a id="supported-media-types"></a>
```

Comment:
```suggestion
```

All headings can be targeted and thus no additional, empty anchor tags are needed